src/main/java/com/nextstep/lotto/store/LottoMaker.java

```diff
@@ -0,0 +1,29 @@
+package com.nextstep.lotto.store;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.nextstep.lotto.lotto.Lotto;
+import com.nextstep.lotto.lotto.LottoNumberGenerator;
+import com.nextstep.lotto.lotto.Lottos;
+
+public class LottoMaker {
```

Comment:
객체를 생성하는 역할을 맡는 클래스를 보통 팩토리 클래스라고 부릅니다.
아래와 같은 이름으로 바꾸면 어떨까요

```suggestion
public class LottoFactory {
```

Reply:
감사합니다 단순히 만든다 의 의미만을 생각해서 클래스 이름을 지었네요
보편적으로 쓰는 용어들(?) 에 대해서도 생각해보아야 겠습니다!